opencti-platform/opencti-graphql/src/modules/notifier/notifier.ts

```diff
@@ -0,0 +1,40 @@
+import { v4 as uuidv4 } from 'uuid';
+import notificationTypeDefs from './notifier.graphql';
+import webhookResolvers from './notifier-resolver';
+import { ENTITY_TYPE_NOTIFIER, StixNotifier, StoreEntityNotifier } from './notifier-types';
+import { ABSTRACT_INTERNAL_OBJECT } from '../../schema/general';
+import type { ModuleDefinition } from '../../schema/module';
+import { registerDefinition } from '../../schema/module';
+import { convertNotifierToStix } from './notifier-converter';
+
+const NOTIFIER_DEFINITION: ModuleDefinition<StoreEntityNotifier, StixNotifier> = {
+  type: {
+    id: 'notifiers',
+    name: ENTITY_TYPE_NOTIFIER,
+    category: ABSTRACT_INTERNAL_OBJECT
+  },
+  graphql: {
+    schema: notificationTypeDefs,
+    resolver: webhookResolvers,
+  },
+  identifier: {
+    definition: {
+      [ENTITY_TYPE_NOTIFIER]: () => uuidv4(),
+    },
+  },
+  attributes: [
+    { name: 'name', type: 'string', mandatoryType: 'internal', multiple: false, upsert: false },
+    { name: 'description', type: 'string', mandatoryType: 'no', multiple: false, upsert: false },
+    { name: 'built_in', type: 'boolean', mandatoryType: 'no', multiple: false, upsert: false },
+    { name: 'notifier_connector_id', type: 'string', mandatoryType: 'internal', multiple: false, upsert: false },
+    { name: 'notifier_configuration', type: 'json', mandatoryType: 'no', multiple: false, upsert: false },
```

Comment:
you can use schemaDef to validate json

Reply:
Not really in that case, because the format change depending on the connector. It seems pretty complex, may be a future update ?